Pod/Classes/Optional Data Sources/VOKPagingFetchedResultsDataSource.m

```diff
@@ -38,54 +39,114 @@ - (void)setupForTriggerDistance:(CGFloat)overscrollTriggerDistance
 {
     self.upAction = upPageActionOrNil;
     self.downAction = downPageActionOrNil;
-    
+
     self.headerView = headerViewOrNil;
     self.footerView = footerViewOrNil;
-    
+
     self.isLoading = NO;
-    
+
     self.triggerDistance = overscrollTriggerDistance;
     self.orginalInsets = self.tableView.contentInset;
-    
+
     [self setupAccessoryViews];
-    
 }
 
 - (void)setupAccessoryViews
 {
     //Attach given views, or generate default views.
     if (!self.headerView) {
-        CGRect headerFrame = (CGRect){0, -DefaultAccessoryHeight, self.tableView.frame.size.width, DefaultAccessoryHeight};
-        self.headerView = [[VOKDefaultPagingAccessory alloc] initWithFrame:headerFrame];
+        self.headerView = [[VOKDefaultPagingAccessory alloc] init];
+        self.headerView.translatesAutoresizingMaskIntoConstraints = NO;
+        [self.headerView addConstraint:[NSLayoutConstraint constraintWithItem:self.headerView
+                                                                    attribute:NSLayoutAttributeHeight
+                                                                    relatedBy:NSLayoutRelationEqual
+                                                                       toItem:nil
+                                                                    attribute:0
+                                                                   multiplier:1
+                                                                     constant:DefaultAccessoryHeight]];
     }
-    
-    self.headerView.frame = (CGRect){0, -self.headerView.frame.size.height, self.headerView.frame.size};
-    [self.tableView addSubview:self.headerView];
-    
+
+    self.tableView.tableHeaderView = self.headerView;
+
+    [self.tableView addConstraint:[NSLayoutConstraint constraintWithItem:self.headerView
+                                                               attribute:NSLayoutAttributeWidth
+                                                               relatedBy:NSLayoutRelationEqual
+                                                                  toItem:self.tableView
+                                                               attribute:NSLayoutAttributeWidth
+                                                              multiplier:1
+                                                                constant:0]];
+    [self.tableView addConstraint:[NSLayoutConstraint constraintWithItem:self.headerView
+                                                               attribute:NSLayoutAttributeLeft
+                                                               relatedBy:NSLayoutRelationEqual
+                                                                  toItem:self.tableView
+                                                               attribute:NSLayoutAttributeLeft
```

Comment:
Is there a difference in this context between `NSLayoutAttributeLeft` and `NSLayoutAttributeLeading`?